### ISSUE TITLE
Jacoco Compatability

### ISSUE THREAD

**Issue by tbadams** [gladed/gradle-android-git-version#41]
Hi, I am attempting to use 'gradle-android-git-version ' for versioning on an Android library project using buildToolsVersion 24.0.0 and gradle version 2.2.3.  The plugin is doing exactly what we want, but unfortunately merely including it in the plugins block causes our unit test and coverage reports to fail with

`java.lang.NoClassDefFoundError: Failed resolution of: Lorg/jacoco/agent/rt/internal_b0d6a23/Offline;`

In the past I've seen error signify [version compatibility problems](https://stackoverflow.com/questions/39195754/java-lang-noclassdeffounderror-failed-resolution-of-lorg-jacoco-agent-rt-inter), but I've tried most reasonable jacoco versions without effect.

Are there any known interactions between the git version plugin and Jacoco? Or similar build tools conflicts?  Thanks!

**Comment by gladed** [comment]
It could be a conflict between the two plugins. I chose `com.android.tools.build:gradle:2.1.3` but you could try forcing it to your version with 

```
configurations.all {
    resolutionStrategy {
        force 'com.android.tools.build:gradle:2.2.3'
    }
}
```

Let me know if this fixes it, if so I'll simply bump the dependency ahead.

**Comment by tbadams** [comment]
Thanks for the suggestion! I gave it a try and still encountered the problem.  If it's not a versioning issue I'm kind of out of ideas.  Jacoco is pretty fragile, so though there's a clear causal link between adding Git Version and the test crash it's possible that it could have more to do with Jacoco than this plugin.

Let me know if you have any further ideas, otherwise I can play with it a little longer and close this out if I can't find anything.  Thanks again.

**Comment by gladed** [comment]
The current Gradle version is 2.3.3 so I'm going to try moving ahead to that. Hopefully it will clear things up...

**Comment by gladed** [comment]
Version 0.3.5 posted.

**Comment by gladed** [comment]
Any luck with the new version, @tbadams ?

**Comment by gladed** [comment]
FYI looked like there was a problem with the 0.3.5 upload. I've re-uploaded so now at 0.3.6.

**Comment by tbadams** [comment]
Sorry for the delay, my work queue was shuffled around.

The issue is completely fixed with version 0.3.6! Unit tests and coverage reports are both working again.  Thank you so much for the update.
